event_webhook_types.go

```diff
@@ -37,6 +37,7 @@ type BuildEvent struct {
 	BuildName         string     `json:"build_name"`
 	BuildStage        string     `json:"build_stage"`
 	BuildStatus       string     `json:"build_status"`
+	BuildCreatedAt    string     `json:"build_created_at"`
```

Comment:
Agree with that. In that case, this should be smoothed out for all attributes of type `*time.Time` in the webhook model. Imo it would be worse to have `string`s and `*time.Time` in the same model.